johnhester/serverless-stack-client
# Initial setup

# Description
- added favicon html & files
- added PT Serif & Open Sans fonts
- installed react bootstrap

## Type of change
# Testing Instructions
see guide at [https://serverless-stack.com/chapters/create-a-new-reactjs-app.html](https://serverless-stack.com/chapters/create-a-new-reactjs-app.html)
# Checklist:
